### PR TITLE
Update feedback.md

### PR DESCRIPTION
**Pull request by Wajeed-msft**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/feedback.md](https://github.com/MicrosoftDocs/msteams-docs/blob/682ea7511408ef1d075657442b45123a390fa9c4/msteams-platform%2Ffeedback.md) | 88 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/13eea90f-e0d8-4e76-8f08-abc3e47ce52a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/feedback.md [hunk 1]
```diff
@@ -22,6 +22,9 @@ Use **Azure Admin Center** or **Microsoft 365 Admin Center** for any **business-
 
 Teams community of developers use Stack Overflow and Microsoft Q&A to connect with other developers to ideate, get clarifications, and submit queries. In addition, you can also use other contacts or sites, depending on the type of community help required to raise issues, limitations, and general questions.
 
+
+📢 Please read our latest [annoucements](https://aka.ms/M365PlatformAnnouncement) and join the conversation with community members and platform engineering!
```
Comment (v-sdhakshina) [inline msteams-platform/feedback.md:26]:
```suggestion
📢 We recommend you go through our latest [annoucements](https://aka.ms/M365PlatformAnnouncement) and join the conversation with both community members and our platform engineering team!
```

**Review by v-sdhakshina — CHANGES_REQUESTED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/feedback.md](https://github.com/MicrosoftDocs/msteams-docs/blob/aa5aaa85a5f548583f5b664ed5d282f884a9343a/msteams-platform%2Ffeedback.md) | 88 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/451e6371-515d-49ff-a771-bcb028f479d4) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/index.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/aa5aaa85a5f548583f5b664ed5d282f884a9343a/msteams-platform%2Findex.yml) | 78 | 44 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/96a8288f-7262-4045-aeed-b11b168d93f7) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/feedback.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6fe460e5cb72797ee7be7b1f5e116c2d67f8027c/msteams-platform%2Ffeedback.md) | 88 | 10 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fbeeaf05-0667-49de-ad0f-7573d5190714) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/index.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/6fe460e5cb72797ee7be7b1f5e116c2d67f8027c/msteams-platform%2Findex.yml) | 80 | 44 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f577be2f-da71-4989-91de-47b6b33018e4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/feedback.md](https://github.com/MicrosoftDocs/msteams-docs/blob/7d07d487a2c117f325f6a5862d7a8c3baf42d69a/msteams-platform%2Ffeedback.md) | 89 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c97c0403-83fc-4719-8271-7f8d867c28c8) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/index.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/7d07d487a2c117f325f6a5862d7a8c3baf42d69a/msteams-platform%2Findex.yml) | 80 | 44 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5537b7d8-3306-411d-ac5d-f16197fa1b80) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-sdhakshina — DISMISSED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/feedback.md](https://github.com/MicrosoftDocs/msteams-docs/blob/17b49f8e6b549509d205821fc0a181f706c6613a/msteams-platform%2Ffeedback.md) | 89 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e14a5f0b-d461-4e5e-94de-79f81687a12c) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/index.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/17b49f8e6b549509d205821fc0a181f706c6613a/msteams-platform%2Findex.yml) | 80 | 44 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3867194c-29e2-4440-97a2-aaaa0ae685c2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**